src/hooks/use-translations.ts

```diff
@@ -1,12 +1,12 @@
-import { useMemo, useTranslator } from "@nebula.js/stardust";
+import { useEffect, useTranslator } from "@nebula.js/stardust";
 import registerLocale from "../locale/src";
 import { ExtendedTranslator } from "../types/types";
 
 export const useTranslations = () => {
   const translator = useTranslator() as ExtendedTranslator;
   const language = translator.language();
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  useMemo(() => registerLocale(translator), [translator, language]);
+
+  useEffect(() => registerLocale(translator), [translator, language]);
```

Comment:
how come you change to useEffect?

Reply:
I did kind of agreed with https://github.com/qlik-oss/sn-pivot-table/pull/251#discussion_r1182177396 and it was nice to get ride of the disabled lint rule. But having giving it some more thought.

It boils down to which `useEffect` finish first, the one that register locale OR the `useEffect` that render the chart, as they are both async. While I do think it's unlikely that registering locale will take more time to finish than rendering the actual chart. We simply do know what `translator.add()` calls does and how much work that requires.

So I will actually reverat back to `useMemo` again.

@a-m-dev